resources/styles/nheko-dark.qss

```diff
@@ -84,12 +84,8 @@ dialogs--Logout,
 dialogs--LeaveRoom,
 dialogs--CreateRoom,
 dialogs--InviteUsers,
-dialogs--JoinRoom {
-    background-color: #383c4a;
-    color: #caccd1;
-}
-
-QListWidget {
```

Comment:
Why was this line deleted?

Reply:
Oops that was accidental. It probably slipped in when I was doing a rebase onto upstream from my previous PR. It has been fixed.